packages/store/src/internal/dispatcher.ts

```diff
@@ -88,11 +96,11 @@ export class InternalDispatcher {
         exhaustMap((ctx: ActionContext) => {
           switch (ctx.status) {
             case ActionStatus.Successful:
-              return of(this._stateStream.getValue());
+              return scheduled([this._stateStream.getValue()], queueScheduler);
```

Comment:
Is `of` deprecated? I've just read Rx's changelog, they didn't fully deprecate `of` factory. `Yes, some overloads of of are deprecated` (for those who used `of` with schedulers)

The `scheduled` factory has more complicated logic. `of` returns a single `new Observable`. `scheduled` factory creates 3 underlying instances of `Subscription` class and instances of `Scheduler` and `Action`. P.S. `Action` class also extends `Subscription`.

[source](https://github.com/ReactiveX/rxjs/blob/b5a2ac9d3576227291238bf36244a54d9739edf6/src/internal/scheduled/scheduleObservable.ts#L9)

P.S. `QueueScheduler` extends `AsyncScheduler` that extends `Scheduler`. `of` is much lightweight and doesn't produce a lot of memory usage just for a single `Observable`.